code/swb_base/ui/SniperScope.cs

```diff
@@ -65,7 +66,7 @@ public override void Tick()
         if (scope == null) return;
 
         // Rotation impact
-        var rightVector = player.EyeRotation.Right * player.Velocity;
+        var rightVector = player.EyeRotation.Right * ent.Velocity;
```

Comment:
Why not use the player variable here instead of ent?

Reply:
Same as above; ISWBPlayer didn't derive from IEntity but it should so I can use ply now instead.